app-backend/api/src/main/scala/user/Routes.scala

```diff
@@ -331,8 +332,33 @@ trait UserRoutes
                   }
                   _ <- campaignsO traverse { campaigns =>
                     campaigns traverse { campaign =>
-                      AnnotationProjectDao
-                        .assignUsersToProjectsByCampaign(campaign.id, usersO)
+                      val campaingRulesO = usersO.map(
+                        users =>
+                          users.foldLeft(List[ObjectAccessControlRule]())(
+                            (acc, user) =>
+                              acc ++ List(
+                                ObjectAccessControlRule(
+                                  SubjectType.User,
+                                  Some(user.id),
+                                  ActionType.View
+                                )
+                            )
+                        )
```

Comment:
I love a `fold` as much as the next person, but isn't a `map` sufficient here? (again, sorry for scalafmt being wrong if you accept this suggestion)

```suggestion
                      val campaingRulesO = usersO.map(
                        users =>
                          users map { user =>
                                ObjectAccessControlRule(
                                  SubjectType.User,
                                  Some(user.id),
                                  ActionType.View
                                )
                              }
```